app/states/texas/validators/texas_flags.py

```
@@ -0,0 +1,53 @@
+from typing import Optional
+from pydantic import BeforeValidator
+from sqlmodel import Field
+from .texas_settings import TECBaseModel
+import funcs.validator_functions as funcs
+
+
+class TECFlagsBase(TECBaseModel):
+    __tablename__ = "tx_flags"
+    __table_args__ = {"schema": "texas"}
```

Ruff (0.14.7)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)